### ISSUE TITLE
how much mAP increase when use IOU as loss for object detection

### ISSUE THREAD

**Issue by 13070151771** [Balupurohit23/IOU-for-bounding-box-regression-in-Keras#1]
hello,how much mAP increase when use IOU as loss for object detection? I want to try this idea too

**Comment by Balupurohit23** [comment]
Hi there, 
The IOU loss function and MSE were used for comparison on ALOV 300++ Dataset with a custom siamese architecture model with the aim to detect an object.
For a brief understanding, I will attach my logs from the best results:
![image O8M13Z](https://user-images.githubusercontent.com/16207266/60738445-f9aa6b80-9f7b-11e9-8710-c53fb6962e03.png)
<p align="center"><b>Fig 1: Training Loss on ALOV 300++ dataset</b></p>

<br>

Furthermore, a separate comparison was done for the model on 20,000 images from Imagenet Dataset for 5 epochs with different loss functions as MSE, IOU, Squared IOU:
![image XJQ23Z](https://user-images.githubusercontent.com/16207266/60740909-139c7c00-9f85-11e9-997a-a7a524a7ed3a.png)
<p align="center"><b>Fig 3: Loss on Imagenet dataset for comparison</b></p>

<br>

The IOU metric was used for accuracy measures: 
<p align="center">
<img src="https://user-images.githubusercontent.com/16207266/60740954-53fbfa00-9f85-11e9-838a-ea8758cbb4b1.png"></p>
<p align="center"><b>Fig 4: IOU metric on all 3 model's validation data for comparison</b></p>
